07-routing-gossip.md

```diff
@@ -285,6 +285,7 @@ The following `address descriptor` types are defined:
          onion service addresses; Encodes:
          `[32:32_byte_ed25519_pubkey] || [2:checksum] || [1:version]`, where
          `checksum = sha3(".onion checksum" | pubkey || version)[:2]`.
+   * `5`: WebSocket port; data = `[2:port]` (length 2)
```

Comment:
Doesn't this also need an interface/IP? Or it's meant to be a sort of modifier?

Reply:
The alternative is to duplicate all the entries, so this is simply a modifier.